app/main/views/services.py

```diff
@@ -162,40 +160,24 @@ def import_service(service_id):
         abort(400, "Cannot update service by PUT")
 
     updater_json = validate_and_return_updater_request()
-    service_json = validate_and_return_service_request(service_id)
-
-    service_data = strip_whitespace_from_data(service_json)
-    service_data = drop_foreign_fields(service_data, ['id'])
-
-    supplier_id = service_data.pop('supplierId')
-    supplier = Supplier.query.filter(
-        Supplier.supplier_id == supplier_id
-    ).first()
-    if supplier is None:
-        abort(400, "Key (supplierId)=({}) is not present".format(supplier_id))
-
-    framework_slug = service_data.get('frameworkSlug')
-    framework = Framework.query.filter(
-        Framework.slug == framework_slug
-    ).first()
-    if framework is None:
-        abort(400, "Key (frameworkSlug)=({}) is not present".format(framework_slug))
-
-    service = Service(service_id=service_id)
-    service.supplier_id = supplier_id
-    service.framework_id = framework.id
-    service.status = service_data.pop('status', 'published')
-    now = datetime.utcnow()
-    service.created_at = service_data.pop('createdAt', now)
-    service.updated_at = service_data.pop('updatedAt', now)
-
-    service.data = service_data
-    validate_service(service, framework=framework)
-    service.data = drop_api_exported_fields_so_that_api_import_will_validate(service.data)
+    service_data = validate_and_return_service_request(service_id)
+
+    framework, lot, supplier = validate_and_return_related_objects(service_data)
+
+    service = Service(
+        service_id=service_id,
+        supplier_id=supplier.supplier_id,
+        lot_id=lot.id,
+        framework_id=framework.id,
+        status=service_data.pop('status', 'published'),
+        created_at=service_data.pop('createdAt', None),
```

Comment:
This is the import endpoint, so we keep the original service create and update timestamps. But it's a good note since there's no need for `.pop` here anymore, I'll remove it.